### PR TITLE
Added tests for Badge component

### PR DESCRIPTION
**Pull request by krolwofficial**
Added tests for Badge component

### CODE REVIEW

File: src/alto-ui/Badge/__tests__/Badge.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Badge from '../Badge';
+
+describe('Badge', () => {
+
+  it('is Badge', () => {
```
Comment (krlakwa) [inline src/alto-ui/Badge/__tests__/Badge.test.js]: Change description for more specific e.g. `should render` or `should render without crashing`

File: src/alto-ui/Badge/__tests__/Badge.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    expect(wrapper.find('.Badge')).toHaveLength(1)
+  });
+
+  it('should return color classNames from color named props', () => {
+    const props = {
+      red: true
+    }
+
+    const wrapper = shallow(<Badge {...props} />);
+    expect(wrapper.html()).toBe('<div class="Badge Badge--red"></div>');
```
Comment (krlakwa) [inline src/alto-ui/Badge/__tests__/Badge.test.js]: try this way:
```
expect(wrapper.hasClass('Badge')).toBe(true);
expect(wrapper.hasClass('Badge--red')).toBe(true);
```

**Comment by barteksiudak** [comment]
There can be added more props by `...props` so check that other props are passed correctly, eg:
```
wrapper.setProps({ id: 'test' });
expect(wrapper.find({ id: 'test'})).toHaveLength(1);
```

File: src/alto-ui/Badge/__tests__/Badge.test.js (hunk 1 above)
Comment (krolwofficial) [reply src/alto-ui/Badge/__tests__/Badge.test.js]: Description changed to `should render without error`

File: src/alto-ui/Badge/__tests__/Badge.test.js (hunk 2 above)
Comment (krolwofficial) [reply src/alto-ui/Badge/__tests__/Badge.test.js]: Fixed

**Pull request merged**